content/en/blog/_posts/2021-11-02-csi-migration-status.md

```diff
@@ -0,0 +1,122 @@
+
+### Bug fixes and feature improvement
+
+We have been able to identify bugs like dangling attachment, garbage collection, incorrect topology label ... and we have fixed all of them to make the feature more healthy.
+
+### Cloud Provider && Cluster Lifecycle Collaboration
+
+We, Sig-Storage, has been working closely with Sig-CloudProvider and Sig-ClusterLifeCycle to provide a better experience for the feature launching.
+
+If you are a user from public cloud provider managed Kubernetes service like GKE, EKS, AKS, there is nothing need to be done from your end. The cloud providers will manage the feature gate and ensure the feature lands correctly.
+
+If you are a user of some other kubernetes distribution services, check the official documentations to get support for the feature. We have communicated with the Sig-ClusterLifeCycle to request early planning for CSI Migration GA and they have been working hard to ensure a smooth experience.
+
+## What is the timeline/status?
+
+The current and targeted releases for each individual driver is shown in the table below:
+
+| Driver            | Alpha         | Beta (in-tree deprecated) | Beta (on-by-default) | GA            | Target "in-tree plugin" removal |
+| ----------------- | ------------- | ------------------------- | -------------------- | ------------- | ------------------------------- |
+| AWS EBS           | 1.14          | 1.17                      | 1.23                 | 1.24 (Target) | 1.26 (Target)                   |
+| GCE PD            | 1.14          | 1.17                      | 1.23                 | 1.24 (Target) | 1.26 (Target)                   |
+| OpenStack Cinder  | 1.14          | 1.18                      | 1.21                 | 1.24 (Target) | 1.26 (Target)                   |
+| Azure Disk        | 1.15          | 1.19                      | 1.23                 | 1.24 (Target) | 1.26 (Target)                   |
+| Azure File        | 1.15          | 1.21                      | 1.23                 | 1.24 (Target) | 1.26 (Target)                   |
+| VSphere           | 1.18          | 1.19                      | 1.23                 | 1.24 (Target) | 1.26 (Target)                   |
+| Ceph RBD          | 1.23          |
+| Portworx          | 1.23          |
+
+The following drivers will not have CSI migration support and will be deprecated and removed from the core Kubernetes.
+
+| Driver    | Deprecated | Code Removal  |
+| ----------| -----------| ------------- |
+| StorageOS | 1.22       | 1.25 (Target) |
+| ScaleIO   | 1.16       | 1.22          |
+| Flocker   | 1.22       | 1.25 (Target) |
+| Quobyte   | 1.22       | 1.25 (Target) |
+
+## What's next?
+
+With more CSI drivers come to Migration GA, we will soon mark the overall CSI Migration feature as GA. We are expecting cloud provider in-tree storage plugins code removal to happen by Kubernetes v1.26.
```

Comment:
Possible rephrase.

With more CSI drivers graduating to GA, we will soon mark the overall CSI Migration feature as GA.